.gitignore

```diff
@@ -1,3 +1,122 @@
-node_modules
```

Comment:
These two weren't deleted--they are just included lower down in the auto-generated lists.